SingularityUI/package.json

```diff
@@ -89,6 +91,12 @@
     "gulp-util": "^3.0.7",
     "mocha": "^3.1.2",
     "nib": "^1.1.0",
+    "node-sass": "^3.8.0",
+    "react-copy-to-clipboard": "^4.2.3",
+    "react-circular-progressbar": "^0.1.3",
+    "react-hot-loader": "^3.0.0-beta.2",
```

Comment:
react-copy-to-clipboard and react-circular-progressbar are the new ones. The others are from hs_staging